pyproject.toml

```diff
@@ -20,6 +20,7 @@ qdrant-client = "^1.9.1"
 pydantic = "^2.7.3"
 openai = "^1.33.0"
 posthog = "^3.5.0"
+chromadb = "^0.5.3"
```

Comment:
Let's not add it to our optional dependency list. Instead, use the try/except block around chroma import and show appropriate error to the user to install it properly.

Reply:
Moreover, Chroma is quite heavy and we don't want to bloat our package with that.

Reply:
Alright will remove it